tox.ini

```diff
@@ -1,5 +1,5 @@
 [tox]
-envlist = py36,py38
+envlist = py35,py36,py38
```

Comment:
Do we want to support 35? We are pretty close to its EOL (September 2020).

Reply:
Doesn't stop it from being default python3 on Xenial though :/

Reply:
(I basically didn't want to have to jump down the rabbit hole of making sure that bcc-tools would work correctly with python3.6 right now)

Reply:
Gotcha